test/render_jsonapi_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class RenderTest < ActionController::TestCase
+  include Roar::Rails::TestCase
+
+  class SingersController < ActionController::Base
+
+    module SingerRepresenter
+      include Roar::JSON::JSONAPI
+      # type :musicians
+      property :name, :as => :title
+    end
+
+    module SingersRepresenter
+      include Representable::JSON::Collection
+      self.represenation_wrap = :musicians
+      items :extend => PersonRepresenter
+    end
+
+    include Roar::Rails::ControllerAdditions
+    include Roar::Rails::ControllerAdditions::Render
+
+    # NOTE: using :json_api will break
+    represents :json, :entity => SingerRepresenter, :collection => SingersRepresenter
```

Comment:
Heh, that's the thing I didn't realize `:collection` was a hack... mostly because I'm not as into the Roar project as you are and you know what should work and when. Thanks for the tip and will definitely keep that in mind to provide broken tests! :)